src/components/ChatLog.jsx

```diff
@@ -0,0 +1,38 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+/* added sender, body, and timeStamp props*/
+const ChatLog = ({ entries, onToggleLike }) => {
+  return (
+    <div className="chat-log">
+      {entries.map((entry) => (
+        <ChatEntry
+          key={entry.id}
+          id={entry.id}
+          sender={entry.sender}
+          body={entry.body}
+          timeStamp={entry.timeStamp}
+          liked={entry.liked}
+          onToggleLike={onToggleLike}
+        />
+      )
+      )}
+    </div>
```

Comment:
`div` is not a semantic element, since we are defining a particular section of the page content, I would consider using `<section>`. Since we are presenting a list of items to a user, `ol` or `ul` would also be options that could make sense.